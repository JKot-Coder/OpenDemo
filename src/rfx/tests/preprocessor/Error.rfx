// :RUN_RFXC -P %STD_OUTPUT% %INPUT%
// #error support

#define FIRST 0

#if FIRST
#error Not this one!
#else

#error This isn't valid!

#endif