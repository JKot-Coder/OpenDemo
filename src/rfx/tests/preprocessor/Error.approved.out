SourceOutput:[
]
Diagnostic:[
    ../src/rfx/tests/preprocessor/Error.rfx(10): error 15900: #error: This isn't valid!
    #error This isn't valid!
     ^~~~~
]
